src/helpers.js

```diff
@@ -36,3 +36,22 @@ export const generateHumanizedTime = (timeRemainingSeconds) => {
   }
   return remainingTime;
 };
+
+// The only information we get on the remaining time on the active exam attempt
+// from the endpoint is the remaining seconds. We need to have a fixed time reference
+// on the time limit to be able to calculate the remaining time accurately.
+export const aggregateActiveAttemptData = (activeAttempt) => {
```

Comment:
This function name `aggregateActiveAttemptData` is confusing. Why use the word `aggregate`? 
This function essentially appends the `timer_ends` attribute to the original json. Why not just call the function `appendTimerEnd`?

Reply:
By ´aggregate´ I meant that I'm adding more data to existing data. I think I've misused the term, maybe something like ´compute´ or ´process´ would have been better. I tend to name functions in a generic way, since this can be used to add more data in the future. I'll rename it as ´appendTimerEnd´ instead as suggested.